.circleci/config.yml

```diff
@@ -265,6 +265,10 @@ commands:
           path: ~/workbench/api/build/test-results/test
           destination: JunitTestResult
           when: always
+      - store_artifacts:
+          path: ~/workbench/api/heapdump.bin
+          destination: oom
+          when: always
```

Comment:
I think the dump file will be generated only when out of memory, which means job has failed. `when: on_fail` is probably better than `always`.